### ISSUE TITLE
GPU binaries are not published on PyPI

### ISSUE THREAD

**Issue by doichanj** [Qiskit/qiskit-aer#2011]
### Informations

- **Qiskit Aer version**: 0.13.1
- **Python version**:
- **Operating system**:

### What is the current behavior?
We released Aer 0.13.1 but only GPU binaries are not published on PyPI.

### Steps to reproduce the problem
Here is a log of the deploy action.

https://github.com/Qiskit/qiskit-aer/actions/runs/6977865439/job/18988325464

The binary packages were built successfully but uploading to PyPI is failed as following.

```
/usr/bin/docker run --name d72fc0718aa8e5c463da379d5693f2a2396_1cf1f8 --label 238d72 --workdir /github/workspace --rm -e "pythonLocation" -e "PKG_CONFIG_PATH" -e "Python_ROOT_DIR" -e "Python2_ROOT_DIR" -e "Python3_ROOT_DIR" -e "LD_LIBRARY_PATH" -e "INPUT_PACKAGES-DIR" -e "INPUT_USER" -e "INPUT_PASSWORD" -e "INPUT_REPOSITORY-URL" -e "INPUT_REPOSITORY_URL" -e "INPUT_PACKAGES_DIR" -e "INPUT_VERIFY-METADATA" -e "INPUT_VERIFY_METADATA" -e "INPUT_SKIP-EXISTING" -e "INPUT_SKIP_EXISTING" -e "INPUT_VERBOSE" -e "INPUT_PRINT-HASH" -e "INPUT_PRINT_HASH" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_ENVIRONMENT" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e "ACTIONS_ID_TOKEN_REQUEST_URL" -e "ACTIONS_ID_TOKEN_REQUEST_TOKEN" -e "ACTIONS_RESULTS_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/qiskit-aer/qiskit-aer":"/github/workspace" 238d72:fc0718aa8e5c463da379d5693f2a2396  "__token__" "" "" "wheelhouse" "" "" "false" ""
Unable to find image '238d72:fc0718aa8e5c463da379d5693f2a2396' locally
docker: Error response from daemon: pull access denied for 238d72, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
```

**Comment by JRGit4UE** [comment]
Any update on that?

**Comment by doichanj** [comment]
We still can not publish GPU package to PyPI https://github.com/Qiskit/qiskit-aer/actions/runs/7551126426/job/20557873811

**Comment by doichanj** [comment]
This is fixed by PR #2034
